projects/aca-shared/src/lib/components/open-in-app/open-in-app.component.scss

```diff
@@ -0,0 +1,15 @@
+.container{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+::ng-deep .mat-dialog-container{
+  padding: 12px;
+  border-radius: 36px;
+  background-color: var(--theme-blue-button-color);
+  color: var(--theme-about-panel-background-color);
+}
+::ng-deep .material-icons{
```

Comment:
ng-deep not needed

Reply:
noted and resolved, ViewEncapsulation.None added